python/lsst/meas/algorithms/detection.py

```diff
@@ -779,6 +785,19 @@ def detectFootprints(self, exposure, doSmooth=True, sigma=None, clearMask=True,
 
         return results
 
+    def removeBadPixels(self, middle):
+        """Set the significance of flagged pixels to zero.
+
+        Parameters
+        ----------
+        middle : `lsst.afw.image.ExposureF`
+            Score or maximum likelihood difference image.
+            The image plane will be modified in place.
+        """
+        badPixelMask = lsst.afw.image.Mask.getPlaneBitMask(self.config.badMaskPlanes)
```

Comment:
There should be no distinction, since the mask planes are defined globally. I try to use the `lsst.afw.image.Mask.getPlaneBitMask()` invocation in general to make that clear.

Reply:
That "global" definition is one of the things that will be changing on DM-32438, and I'm trying to catch uses of it like this ahead of time. It's held up by a house of cards, and isn't as guaranteed to be global as one might hope. Too late for this ticket now, but please use the `mask` instance call in the future.